### DISCUSSION TITLE
Running 2 RU's with a single SRSRan Software

### DISCUSSION THREAD

**Discussion by ajesh-pk** [srsran/srsRAN_Project#750] [General Help]
We have two VVDN RU's where one is of N77 and the other is N78 Band, we tested both RU's by changing the RU MAC address and ARFCN in gNB config, Both are working fine.
NOw, 
how can I merge both the configs so that gNB can drive the both radios, provided we have a 25G FH interface, so it results in a Multi cell setup.

**Comment by pgawlowicz** [comment]
You can run a disaggregated setup with two DUs each connected to its own RU.

Please check here: https://docs.srsran.com/projects/project/en/latest/tutorials/source/cu_du_split/source/index.html

**Reply by pgawlowicz** [reply]
@ajesh-pk any update on this issue?
